journald/journald.cpp

```diff
@@ -75,6 +83,27 @@ class JournaldLoggerProcess : public Process<JournaldLoggerProcess>
       std::strcpy((char*) entries[i].iov_base, entry.c_str());
```

Comment:
I left these as-is because this initialization happens only once at the start.  Within the loop, there is only one copy (which copies a pointer), and this only runs in the `journald` mode